corehq/apps/accounting/templates/accounting/partials/new_stripe_card_template.html

```diff
@@ -78,7 +78,7 @@ <h4 class="modal-title">{% trans "Add New Credit Card" %}</h4>
                                     <input type="checkbox"
                                            data-bind="checked: agreedToPrivacyPolicy, disable: isProcessing" />
                                     {% blocktrans %}
-                                        I agree to Dimagi's <a href="http://www.dimagi.com/policy" target="_blank">Privacy Policy</a>.
+                                        I agree to Dimagi's <a href="http://www.dimagi.com/terms/#privacy" target="_blank">Privacy Policy</a>.
```

Comment:
fyi, the new policy link will be `https://www.dimagi.com/terms/latest/privacy/`

Reply:
Should the links in this PR to `https://www.dimagi.com/terms` also be updated? Looks like that link now goes to the old privacy policy.